blocks/library/paragraph/editor.scss

```diff
@@ -2,6 +2,11 @@
 	background: white;
 }
 
+// Don't show white background when a nesting parent is selected
```

Comment:
What's the reasoning for the white background in the first place?

Reply:
I actually can't recall. Hmm. @youknowriad?

Reply:
Looks like it was introduced here: https://github.com/WordPress/gutenberg/commit/775428e4778f406f1b8c815b54518d2350edc49c#diff-c2685bc3733aece65a4ed211668982e0R2

So it's probably related to the ability to colorize paragraph backgrounds. But removing it doesn't seem to make a difference. @jorgefilipecosta any thoughts?

Reply:
Seems we shouldn't want to assume a white background, especially if the plan is to [support themes to define their own editor styles](https://youtu.be/XOY3ZUO6P0k?t=3032).

Reply:
The reason why we set a white background is related to the contrast verification. If the background was never set getComputedStyles returns back background (if I remember correctly). So adding a background fixes the problem. If themes change the background it also works well because as long as a background is set we are able to retrieve it. The problem happens when no background is set. As we get back background color when background is not set we cannot differentiate and assume is white background because in fact it may be set to black. I will try to research this in more detail maybe there is a way to detect when the background is not set and in that case assume white.
Replicate from comment https://github.com/WordPress/gutenberg/pull/5273#discussion_r176880358.

Reply:
But this only works if the style is explicitly set, i.e. a theme would need to target and override the `.editor-block-list__block:not( .is-multi-selected ) .wp-block-paragraph` selector (or selector of equivalent specificity). And the contrast checker would otherwise falsely report contrast issues on the basis of a white background? i.e. if theme actually had black background where a white text would be very contrasting, it would be reported by the editor as an issue.

I'm wondering if it's really worth the trouble to assume a default value (vs. disabling until a background color is known / set).

Alternatively, could we recurse up through parent elements until we find one where background color is not the default value?

Reply:
I think we can remove the background color, for now, as it is causing us troubles so we can advance.
After, I really like the idea, of recursively iterating up to the parent elements. And I will try to explore it. If we find it not a good solution than we disable contrast checker for colors not explicitly set.